recipe/patch_yaml/nbconvert.yaml

```diff
@@ -104,3 +104,11 @@ then:
   - replace_constrains:
       old: pandoc?( *)
       new: pandoc >=1.12.1,<3.0.0
+---
+if:
+  name_in: [nbconvert, nbconvert-core]
+  version_lt: "7.1.0"
+  version_ge: "7.0.0"
```

Comment:
`lxml` was introduced in 7.0.0 and removed in 7.1.0.

Reply:
Introduction: https://github.com/jupyter/nbconvert/commit/7100b2b7b12d501290f3142a791db9bec1fa1b32
Removal: https://github.com/jupyter/nbconvert/commit/b40bb13a44d5632f20e8bb6d13049dccb6dbfa75